docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - .:/app

```

Comment:
Mounting the entire current directory into the container can lead to potential issues. Consider specifying only the necessary directories or files to avoid overwriting container files or including unnecessary files.